pkg/cmd/pulumi/preview.go

```diff
@@ -445,6 +454,7 @@ func newPreviewCmd() *cobra.Command {
 					GeneratePlan:   hasExperimentalCommands() || planFilePath != "",
 					Experimental:   hasExperimentalCommands(),
 					AttachDebugger: attachDebugger,
+					Autonaming:     autonamer,
```

Comment:
Feels to me like this should either be `Autonamer: autonamer` or `Autonaming: autonaming` -- we should have one name for whatever this does ("autonamer" feels right?)